stdlib/public/core/FloatingPointTypes.swift.gyb

```diff
@@ -1354,7 +1354,9 @@ extension ${Self} : _ExpressibleByBuiltinIntegerLiteral, ExpressibleByIntegerLit
   }
 }
 
+% if bits != 80:
```

Comment:
What's the goal of adding the new check here?

Reply:
I was just syntactically annoyed that the existing gyb implementation outputs:
#if !os(windows) && (arch(i386) || arch(x86_64))
//code
#if !os(windows) && (arch(i386) || arch(x86_64))
//code
#endif
#endif

Reply:
It's difficult to see what's going on here in GitHub diff form, I'll need to look at this more carefully.